--- origsrc/poedit-1.5.7/src/fileviewer.cpp	2013-07-05 12:25:25.000000000 -0500
+++ src/poedit-1.5.7/src/fileviewer.cpp	2013-12-09 22:34:24.792009100 -0600
@@ -286,7 +286,11 @@ void FileViewer::ShowReference(wxString
         linenum = 0;
 
     m_text->SetReadOnly(false);
+#if wxCHECK_VERSION(2,9,0)
     m_text->SetValue(data);
+#else
+    m_text->SetText(data);
+#endif
     m_text->SetReadOnly(true);
 
     m_text->MarkerDeleteAll(1);
